bitnami/clickhouse/templates/configd-configmap.yaml

```
@@ -133,4 +133,24 @@ data:
         </prometheus>
     </yandex>
   {{- end }}
+  {{- if not .Values.sampling.enabled }}
+  08-sampling.xml: |
+    <yandex>
+      <asynchronous_insert_log remove="1"/>
+      <asynchronous_metric_log remove="1"/>
+      <error_log remove="1"/>
+      <event_log remove="1"/>
+      <latency_log remove="1"/>
+      <metric_log remove="1"/>
+      <query_log remove="1"/>
+      <query_metric_log remove="1"/>
+      <query_thread_log remove="1"/>
+      <query_views_log remove="1"/>
+      <part_log remove="1"/>
+      <text_log remove="1"/>
+      <trace_log remove="1"/>
+      <opentelemetry_span_log remove="1"/>
+      <processors_profile_log remove="1"/>
+    </yandex>
```

Please follow the same indentation used on previous files:
```suggestion
    <yandex>
        <asynchronous_insert_log remove="1"/>
        <asynchronous_metric_log remove="1"/>
        <error_log remove="1"/>
        <event_log remove="1"/>
        <latency_log remove="1"/>
        <metric_log remove="1"/>
        <query_log remove="1"/>
        <query_metric_log remove="1"/>
        <query_thread_log remove="1"/>
        <query_views_log remove="1"/>
        <part_log remove="1"/>
        <text_log remove="1"/>
        <trace_log remove="1"/>
        <opentelemetry_span_log remove="1"/>
        <processors_profile_log remove="1"/>
    </yandex>
```